commands/root.go

```diff
@@ -53,6 +55,10 @@ func decorateRootCommand(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringVarP(&cluster, "cluster-name", "n", "", "name of the cluster that contains the instances.")
 	cmd.PersistentFlags().StringVar(&token, "cloud-token", "", "your Hazelcast Cloud token.")
 	cmd.CompletionOptions.DisableDefaultCmd = true
+
+	if mode := os.Getenv("MODE"); strings.EqualFold(mode, "dev") { // This is used to generate completion scripts
```

Comment:
It's OK to use env variables for this for the prototype, but we may consider something like build tags later.

Reply:
Sounds good 👍

Reply:
Is there a reason we consider `MODE` to be case insensitive?

Reply:
I thought it would be friendlier for user, but I can make it only uppercase. Is it better?